package.json

```diff
@@ -11,28 +11,47 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build && tsc && node copy-types.js",
+    "build:css": "tailwindcss -m -i ./src/tailwind-entry.css -o ./dist/styles.css",
+    "preview": "vite preview",
+    "storybook": "concurrently \"pnpm storybook:css\" \"storybook dev -p 6006\"",
+    "storybook:css": "tailwindcss -w -i ./src/tailwind-entry.css -o ./src/index.css",
+    "build-storybook": "concurrently \"pnpm build-storybook:css\" \"storybook build\"",
+    "build-storybook:css": "tailwindcss -m -i ./src/tailwind-entry.css -o ./src/index.css",
     "prepublishOnly": "pnpm build",
     "prepare": "husky install"
   },
   "keywords": [],
   "author": "",
   "license": "GPL-3.0",
   "devDependencies": {
+    "@chromatic-com/storybook": "3.2.2",
     "@commitlint/cli": "^19.6.0",
     "@commitlint/config-conventional": "^19.6.0",
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/github": "^11.0.1",
     "@semantic-release/npm": "^12.0.1",
+    "@storybook/addon-essentials": "8.4.6",
+    "@storybook/addon-interactions": "8.4.6",
+    "@storybook/addon-onboarding": "8.4.6",
+    "@storybook/addon-themes": "^8.4.6",
+    "@storybook/blocks": "8.4.6",
+    "@storybook/manager-api": "^8.4.6",
+    "@storybook/react": "8.4.6",
+    "@storybook/react-vite": "8.4.6",
+    "@storybook/test": "8.4.6",
+    "@storybook/theming": "^8.4.6",
     "@types/node": "^22.9.3",
     "@types/react": "^18.3.12",
     "@vitejs/plugin-react": "^4.3.3",
     "autoprefixer": "^10.4.20",
+    "concurrently": "^9.1.0",
     "husky": "^9.1.7",
     "postcss": "^8.4.49",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "semantic-release": "^24.2.0",
+    "storybook": "8.4.6",
```

Comment:
Im getting an error about an unmet peerdepedency with storybook. Seems to require 8.4.7